evm_loader/program/src/executor/precompile_extension/call_solana.rs

```diff
@@ -24,9 +27,12 @@ use solana_program::{
 // "cd2d1a3a": "getExtAuthority(bytes32)"
 // "30aa81c6": "getPayer()",
 
+// "09c5eabe": "execute(bytes)"
 // "c549a7af": "execute(uint64,bytes)",
 // "32607450": "executeWithSeed(uint64,bytes32,bytes)",
+// "260e40bc": "executeWithSeed(bytes32,bytes)",
 // "aeed7f1e": "execute(uint64,(bytes32,(bytes32,bool,bool)[],bytes))",
```

Comment:
Overload without lamports for `execute(uint64,(bytes32,(bytes32,bool,bool)[],bytes))`?